thirdparty/CMakeLists.txt

```diff
@@ -130,7 +130,7 @@ if(NGRAPH_PDPD_FRONTEND_ENABLE OR NGRAPH_ONNX_IMPORT_ENABLE)
         if(VERBOSE_BUILD)
             set(Protobuf_DEBUG ON)
         endif()
-        find_package(Protobuf REQUIRED)
+        find_package(Protobuf 3.9.0 REQUIRED)
```

Comment:
should this be inside `if(NGRAPH_USE_PROTOBUF_LITE)`?

Reply:
we have submodule 3.9.2 right now, I think we should have commonly aligned version

CC @tsocha could you please advise?